setup.py

```diff
@@ -17,12 +17,12 @@
 
 
 setup(
-    name='glyphs2ufo',
+    name='glyphsLib',
     version='1.0',
-    packages=['glyphs2ufo'],
+    packages=['glyphsLib'],
     entry_points={
         "console_scripts": [
-            "glyphs2ufo = glyphs2ufo.__main__:main"
+            "glyphsLib = glyphsLib.__main__:main"
```

Comment:
That makes sense. Could we just do `glyphs2ufo = glyphsLib.__main__:main`? Or, rename `__main__.py` -> `glyphs2ufo.py`?

On the other hand, there has been interest in adding UFO->Glyphs functionality to this library, which would probably also be an option in the console script. Not sure when that would happen though.